tsconfig.json

```diff
@@ -10,11 +10,12 @@
       /* Bundler mode */
       "moduleResolution": "bundler",
       "allowImportingTsExtensions": true,
+      "strictFunctionTypes": true,
       "resolveJsonModule": true,
       "isolatedModules": true,
       "noEmit": true,
       "jsx": "react-jsx",
-      "noImplicitAny": false,
+      "noImplicitAny": true,
```

Comment:
im surprised we didnt have this already here

Reply:
The repo was a `.js` app at first, so the typescript migration was slower